### DISCUSSION TITLE
Loss functions  `loss=virials` vs. `loss=stress`

### DISCUSSION THREAD

**Discussion by hwsheng** [ACEsuit/mace#222] [General]
I'm seeking some clarification regarding the loss functions implemented in MACE, specifically the differences between `loss=virials` and `loss=stress`. My understanding is that these two are not interchangeable. For `loss=virials`, does the framework utilize per-atom virials within the loss function? 

Regarding `loss=stress`, considering that stress is an intensive property, the concept of per-atom stress as indicated in the loss.py file seems somewhat unconventional. Could you please provide some insight into this?

Thank you for your assistance in understanding these aspects of MACE.

**Comment by ilyes319** [comment]
Sorry for the late reply.
Indeed, the two are not interchangeable. Stress and virials represent the same thing, but if you have the virials format in your data, you the virials loss, and if you have stress, use the stress format.

That is a good point on the per-atom stress. I might change it, thank you.

**Comment by hwsheng** [comment]
Many thanks for the clarification!
Howard Sheng
